TechTeam.md

```diff
@@ -1,32 +1,44 @@
 # Tech-Team, Technology Students' Gymkhana
 
-Tech-Team is the group that will be maintaing all the existing technical projects and brainstorming on new ideas and building applications and software for the betterment of the institute.
+Tech-Team is the group that will be maintaing all the existing technical projects and brainstorming on new ideas and building applications and software for the betterment of the institute. Ideally, the tech-team consists of 4-6 members, in their 2nd year but neither is a hard requirement
 
 ## Responsibilities
 
 The responsibilities of the organisation are as follows along with current maintainers of the project.
 
-- Maintaining ApnaInsti Backend.
-- Maintaining ApnaInsti WebApp.
-- Maintaining Institute Android Platform.
+- Maintaining the Gymkhana Website
+
+- Maintaining the Gymkhana Blog Server
+
 - Perform and execute technical ties with other socities.
-- Manage and update Gymkhana website.
-- Some members will be alloted to maintaining the opensource and server assets of MetaKGP along with MFTP, MFQP and more.
 
-NOTE: As we move forward and create new projects, this document should be updated.
+- Manage and update archival data, and results of various events, such as InterIIT and GC
+
+    
+
+NOTE: As we move forward and create new projects, this document should be updated
 
 ## Formation
 
 The Tech-Team should be formed via the below process by the present Technology Coordinator for his tenure. The previous members of the team can voluntarily contribute. They MUST get added to Alumni team of the organisation on Github.
 
-- A form should be circulated a fortnight before end of the summer breaks to get in entries. An example form for the same is [here](https://shivamkumarjha.typeform.com/to/deomWG). Sufficient time should be given to candidates in order to prepare CV in case they don't have one yet.
-- Interviews should be conducted within the first 15 days of institute reopening after selection on the basis of form responses. All students whose CVs (and responses) are fit for the team shall be invited.
-- From the interview, at max 10 members (preferable lesser) should be chosen. This number can be stretched or squeezed at the discretion of the present (and former) Tech Coordinator.
-- Results should be published via the GymKhana page (and also mail to be sent to new members) within a week of the interviews.
+- A form should be circulated after the nomination of the tech coordinator. An example form for the same is [here](https://forms.gle/9Du4xXSurx6C7dMz8). Ask the PR Chair to create a FB post, with content similar to the one in `Posts/TechTeamFB.md`. Sufficient time should be given to candidates in order to prepare CV in case they don't have one yet.
```

Comment:
```suggestion
- A form should be circulated after the nomination of the tech coordinator. An example form for the same is [here](https://forms.gle/9Du4xXSurx6C7dMz8). Ask the PR Chair to create a FB post, with content similar to the one in `Posts/TechTeamFB.md`. Sufficient time should be given to candidates in order to prepare a CV in case they don't have one yet.
```